Source/IntegrationTests/TestFiles/LitTests/LitTest/c++/arrays.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %testDafnyForEachCompiler --refresh-exit-code=0 "%s" -- --relax-definite-assignment --spill-translation --unicode-char:false
```

Comment:
Why this change?

Reply:
Just cleanup. The test does not need that option.